packages/flutter/lib/src/cupertino/scrollbar.dart

```diff
@@ -183,6 +232,28 @@ class _CupertinoScrollbarState extends State<CupertinoScrollbar> with TickerProv
         ..color = CupertinoDynamicColor.resolve(_kScrollbarColor, context)
         ..padding = MediaQuery.of(context).padding;
     }
+    WidgetsBinding.instance.addPostFrameCallback((Duration duration) {
+      if (widget.displayAlways) {
+        assert(widget.controller != null);
+        // Wait one frame and cause an empty scroll event.  This allows the
+        // thumb to show immediately when displayAlways is true.  A scroll
+        // event is required in order to paint the thumb.
+        widget.controller.position.didUpdateScrollPositionBy(0);
+      }
+    });
+  }
+
+  @override
+  void didUpdateWidget(CupertinoScrollbar oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if (widget.displayAlways != oldWidget.displayAlways) {
+      assert(widget.controller != null);
```

Comment:
What about if displayAlways is changing from true to false, is this assert still valid?

Reply:
Thanks, I think that logic looks right 👍